<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>860</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>860</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>860</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preference Menu (User)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
stop:0 #4D9E63, stop:1 #5A99D9);


}
QPushButton {
    background-color: #2196F3;
    border-radius: 16px;
    border: 2px solid #1976D2;
    padding: 10px 20px;
    font-size: 28px;
    color: white;
    box-shadow: 0 4px #1976D2;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #42A5F5;
    box-shadow: 0 6px #1976D2;
    transform: translateY(-4px);
}

QPushButton:pressed {
    background-color: #1E88E5;
    box-shadow: 0 2px #1976D2;
    transform: translateY(2px);
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton_Exit_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>730</y>
     <width>145</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: #1565C0;
    border-radius: 8px;
    border: 2px solid #1976D2;
    padding: 10px 20px;
    font-size: 22px;
    color: white;
    box-shadow: 0 4px #1976D2;
    transition: all 0.3s ease;
}
QPushButton:hover {
    background-color: #42A5F5;
    box-shadow: 0 6px #1976D2;
    transform: translateY(-4px);
}

QPushButton:pressed {
    background-color: #1E88E5;
    box-shadow: 0 2px #1976D2;
    transform: translateY(2px);
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-10</y>
     <width>1031</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    font: bold 32px &quot;Verdana&quot;;
    color: #004080;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
stop:0 #C2D9F0, stop:1 #85B1CF);

    text-shadow: 2px 2px 3px #996600;
    border-radius: 16px;
}
</string>
   </property>
   <property name="text">
    <string>PREFERENCE MENU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_applications">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Applications</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mentorMeeting">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mentor Meeting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_interviews">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>230</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Interviews</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
